electron/home.js

```diff
@@ -208,14 +242,13 @@ async function moveFile(oldPath, newPath) {
     });
 }
 
-
 saveAudioBlob = async function (audioBlobToSave, fPath) {
     print_both(`Trying to save: ${fPath}`);
 
-    //move old audio file
-    let newFileName = new SimpleDateFormat("yyyy-MM-dd hh-mm-ss'.wav'").format(new Date());
-    let newPath = audio_archive_path + newFileName;
-    await moveFile(fPath, newPath);
+    // //move old audio file
```

Comment:
// // -> //

Reply:
Valid concer, was a comment that got commented. I think we can approve for now though as won't create an other issues and will likely be removed once you have your archival logic in place. Or it may be uncommented and reworked as a means to move the recording.wav in the case that classification errored out before classification script completion.

Reply:
In general we have more code cleanup to do. There are a few things that need renamed. I tried to tackle most instances of code formatting / blank line issues in this PR.